src/state/hooks.ts

```diff
@@ -0,0 +1,39 @@
+import { useMemo } from 'react'
```

Comment:
why this file is not in `hooks` folder ?

Reply:
moved